torchvision/utils.py

```diff
@@ -2,26 +2,24 @@
 import pathlib
 import torch
 import math
+import warnings
 import numpy as np
 from PIL import Image, ImageDraw
 from PIL import ImageFont
 
 __all__ = ["make_grid", "save_image", "draw_bounding_boxes"]
 
-irange = range
-
 
 def make_grid(
     tensor: Union[torch.Tensor, List[torch.Tensor]],
     nrow: int = 8,
     padding: int = 2,
     normalize: bool = False,
-    range: Optional[Tuple[int, int]] = None,
+    value_range: Optional[Tuple[int, int]] = None,
     scale_each: bool = False,
     pad_value: int = 0,
 ) -> torch.Tensor:
```

Comment:
Why not add a `**kwargs` and do exactly the same thing that you do for `save_image()`? It will bring backwards compatibility and offer a warning to the user.